backend/src/test/kotlin/org/loculus/backend/controller/files/GetFilesEndpointTest.kt

```diff
@@ -38,7 +38,7 @@ class GetFilesEndpointTest(
         )
 
         filesClient.getFile(data.accession, data.version, "myFileCategory", "hello.txt")
-            .andExpect(status().isForbidden())
+            .andExpect(status().isUnauthorized())
```

Comment:
do we have a test for the Forbidden case?

Reply:
I don't know - but the error code was wrong so this just fixes that